hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheGetInvalidationMetaDataOperation.java

```diff
@@ -144,6 +151,22 @@ public void readData(ObjectDataInput in) throws IOException {
         }
     }
 
+    private List<Integer> getOwnedPartitions() {
+        IPartitionService partitionService = getNodeEngine().getPartitionService();
+        Map<Address, List<Integer>> memberPartitionsMap = partitionService.getMemberPartitionsMap();
+        return memberPartitionsMap.get(getNodeEngine().getThisAddress());
```

Comment:
Sure, I've aligned the NPE check with your PR here: https://github.com/hazelcast/hazelcast/pull/11200